.github/workflows/system-tests.yml

```diff
@@ -160,102 +160,102 @@ jobs:
           docker load < images_artifacts/agent_${{ github.sha }}.tar.gz
 
       - name: Run DEFAULT
-        if: (needs.needs-run.outputs.outcome == 'success' || github.event_name == 'schedule') && matrix.scenario == 'other'
+        if: always() && (needs.needs-run.outputs.outcome == 'success' || github.event_name == 'schedule') && matrix.scenario == 'other'
```

Comment:
Small improvment : If any of precedent steps fails (docker load, build ...), no scenario will be a success, and the output will be very spammed.

The way I solved that in system tests is : 

1. [add an `id`](https://github.com/DataDog/system-tests/blob/main/.github/workflows/run-end-to-end.yml#L99) at the very last step before the first  `./run.sh`
2. on each `if`, [add a success condition on this very last step](https://github.com/DataDog/system-tests/blob/main/.github/workflows/run-end-to-end.yml#L107) (in this link, the condition is `teps.build.outcome == 'success'`)

With that, you won't run any scenario if one of the preparation step is failing.